WEB_SERVICE/WEB/_CRITICAL/EXPRESSION_LANGUAGE_INJECTION/recommendation.md

```diff
@@ -0,0 +1,28 @@
+To secure the application against Expression Language Injection (EL Injection), consider the following recommendations:
+
+- __Avoid Direct User Input Use__: Whenever possible, avoid directly using user inputs in EL expressions. Instead, prefer a whitelist approach where only predefined, safe values are allowed to be used in EL expressions.
+
+
+- __Input Validation__: Validate and sanitize user inputs before using them in EL expressions. Implement strict validation to accept only expected data types and patterns.
+
+
+- __Context-Specific Encoding__: Use encoding functions provided by your framework or libraries (e.g., \<c:out> in JSP, fn:escapeXml() in JSTL) to ensure context-aware output encoding. This prevents the interpretation of user inputs as code.
+
+### Example
+
+=== Java
+  ```java
+  @RequestMapping(value="/")
```

Comment:
What is the diff between the code in the example and this one ?

Reply:
I've changed the example